packages/trpc/server/auth-router/schema.ts

```diff
@@ -26,3 +26,9 @@ export const ZSignUpMutationSchema = z.object({
 export type TSignUpMutationSchema = z.infer<typeof ZSignUpMutationSchema>;
 
 export const ZVerifyPasswordMutationSchema = ZSignUpMutationSchema.pick({ password: true });
+
+export const ZDeleteAccountMutationSchema = z.object({
+  email: z.string().email(),
```

Comment:
Would we need this? Presumably it can be argless mutation and we get the user from the context value, this means there's no chance of accidentally deleting someone elses account.

We would probably also have a version in the admin router later on for us to offboard users who don't wish to delete their own account.

Reply:
Right, yeah, I'll go ahead without this